docs/basics/update-methods.md

```diff
@@ -214,8 +214,10 @@ the annotation. For example, the following would create a branch named
 `image-updater-foo/bar-1.1` based on `main` in the event an image with
 the name `foo/bar` was updated to the new tag `1.1`.
 
+Helm will try to consume this template unless you escape the opening and closing brackets.
+
 ```yaml
-argocd-image-updater.argoproj.io/git-branch: main:image-updater{{range .Images}}-{{.Name}}-{{.NewTag}}{{end}}
+argocd-image-updater.argoproj.io/git-branch: main:image-updater{{"{{"}}range .Images{{"}}"}}-{{"{{"}}.Name{{"}}"}}-{{"{{"}}.NewTag{{"}}"}}{{"{{"}}end{{"}}"}}
 ```
```

Comment:
Would it make sense to keep the original form for non-helm cases, and also for easy viewing of the real value?  We can add the helm-escaped form for helm cases.